pkg/distributor/distributor.go

```diff
@@ -445,6 +445,7 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 
 // newRingAndLifecycler creates a new distributor ring and lifecycler with all required lifecycler delegates
 func newRingAndLifecycler(cfg RingConfig, instanceCount *atomic.Uint32, logger log.Logger, reg prometheus.Registerer) (*ring.Ring, *ring.BasicLifecycler, error) {
+	reg = prometheus.WrapRegistererWithPrefix("cortex_", reg)
```

Comment:
This changes the metric names for the KV client used by the distributor. Two comments:
1. Can you double check what's the impact on dashboards and alerts?
2. This should be mentioned in the CHANGELOG entry too

Reply:
> Can you double check what's the impact on dashboards and alerts?

I checked it, and we always query `cortex_kv_request_duration_seconds_*` in dashboards and alerts. This means that this is a fix, not a regression, which is good.

However, querying the metric name `kv_request_duration_seconds` (without the `cortex_` prefix) I've found it's also exposed by the query-scheduler.

**Suggestion**: I suggest to revert this change from this PR (keeping the "broken" version) and open a separate PR to fix it both here in the distributor and query-scheduler, so that the change is better tracked in the commits log.

Reply:
Reverted. I'll open a separate PR to fix for distributors and query-scheduler.